Announcements week 42 (#468)
giantswarm/releases: announcements/2020-Week-42/announcement.md

# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:

## Giant Swarm Catalog (Managed Apps)

1. [NGINX Ingress Controller app v1.10.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1100---2020-10-07) upgrades ingress-nginx-controller from `v0.35.0` to `v0.40.2` improving performance and reliability.

> 2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream.)

Reviewer: ```suggestion
2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream. SEs are working with customers to prepare.)
```

> 2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream. SEs are working with customers to prepare.)

Reviewer: @ced0ps any idea on the value this upgrade brings?

Reviewer 2: First major release of `kong-ingress-controller` with:
- https://github.com/kong/kubernetes-ingress-controller/compare/0.10.0...1.0.0
- https://github.com/kong/kubernetes-ingress-controller/compare/0.9.1...0.10.0

Fixes of the chart:
- Removed the status field from the TCPIngress CRD.
- Changes to annotation block generation prevent incorrect YAML indentation when specifying annotations via command line arguments to Helm commands. (#200)

Improvements:
- The chart now adds namespace information to manifests explicitly. This simplifies workflows that use helm template.

> 3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0.

Reviewer: ```suggestion
3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0. It also makes required changes related to the upstream chart name change from `prometheus-operator` to `kube-prometheus`
```

> 3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0. It also makes required changes related to the upstream chart name change from `prometheus-operator` to `kube-prometheus`

Reviewer: @ced0ps are there any known problems w the rename? :) cc @omegas27 for creating the issue on EOL of old chart

Reviewer 2: No, this is limited to the templating of the chart itself. We've opted to not rename the app to avoid any confusion for now.

> 3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0. It also makes required changes related to the upstream chart name change from `prometheus-operator` to `kube-prometheus`

Reviewer: ```suggestion
3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0. It also makes required changes related to the upstream chart name change from `prometheus-operator` (EOL in Nov 2020) to `kube-prometheus`
```

# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:

## Giant Swarm Catalog (Managed Apps)

1. [NGINX Ingress Controller app v1.10.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1100---2020-10-07) upgrades ingress-nginx-controller from `v0.35.0` to `v0.40.2` improving performance and reliability.

> 2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream. SEs are working with customers to prepare.)

Reviewer 3: ```suggestion
2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream. Solution engineers are working with customers to prepare.)
```

> 3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0. It also makes required changes related to the upstream chart name change from `prometheus-operator` (EOL in Nov 2020) to `kube-prometheus`

Reviewer 4: Was there a change in the prometheus operator version?

Reviewer: in the linked [changelog](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15), it says, "prometheus-operator v0.38.1 -> v0.42.1"

that's a good point... Is that the main thing customers would be looking for? If yes, then that should be included here... will talk w team

> 

Reviewer: ```suggestion
4. [Cert Manager v2.3.0](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#230---2020-10-02) brings upstream v1.0.2, the first stable release.
```

@glitchcrab

Reviewer 5: :shipit:

> 4. [Cert Manager v2.3.0](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#230---2020-10-02) brings the first stable release from upstream, v1.0.2.

Reviewer: ```suggestion
4. [Cert Manager v2.3.0](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#230---2020-10-02) brings the first stable release from upstream, v1.0.2.

```

# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:

## Giant Swarm Catalog (Managed Apps)

1. [NGINX Ingress Controller app v1.10.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#1100---2020-10-07) upgrades ingress-nginx-controller from `v0.35.0` to `v0.40.2` improving performance and reliability.

> 2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream. Solution engineers are working with customers to prepare.)

Reviewer: ```suggestion
2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream. Solution Engineers are working with customers to prepare.)
```

> 2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. (Note: Contains breaking changes from upstream. Solution Engineers are working with customers to prepare.)

Reviewer: ```suggestion
2. [Kong v1.0.0](https://github.com/giantswarm/kong-app/blob/master/CHANGELOG.md#100---2020-10-13) upgrades to upstream chart v1.11.0. It includes the first major release of kong-ingress-controller. (Note: Contains breaking changes from upstream. Solution Engineers are working with customers to prepare.)
```

> 3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0. It also makes required changes related to the upstream chart name change from `prometheus-operator` (EOL in Nov 2020) to `kube-prometheus`

Reviewer: ```suggestion
3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0. It also adapts global templating variables to work with upstream chart location and name change from `prometheus-operator` (EOL in Nov 2020) to `kube-prometheus-stack`
```

> 3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades to upstream chart v10.1.0. It also adapts global templating variables to work with upstream chart location and name change from `prometheus-operator` (EOL in Nov 2020) to `kube-prometheus-stack`

Reviewer: ```suggestion
3. [Prometheus Operator v0.4.0](https://github.com/giantswarm/prometheus-operator-app/blob/master/CHANGELOG.md#040---2020-10-15) upgrades prometheus-operator, prometheus, alertmanager, etc. It also adapts global templating variables to work with upstream chart location and name change from `prometheus-operator` (EOL in Nov 2020) to `kube-prometheus-stack`
```
